psychopy/tests/test_misc/test_event.py

```diff
@@ -226,6 +227,20 @@ def setup_class(self):
         self.win = Window([128,128], monitor=mon, winType='pyglet', pos=[50,50], autoLog=False)
         assert pygame.display.get_init() == 0
 
+    def test_shutdown_key(self):
+        key = 'escape'
```

Comment:
Sorry, too busy to clone and test this myself now, but wouldn't this fail? I.e. setting it to a string instead of a list would cause `if thisKey in prefs.general['shutdownKeys']:` in `_onPygletKey` to loop through characters rather than list items. In other words: shouldn't the API here only allow to set `prefs.general['shutdownKeys']` to be a list? Perhaps this should be tested explicitly on assignment. Haven't played with the `psychopy.preferences` before, but looking at it, there's a generic setter which makes such a check convoluted. So we'll probably just leave it for now.